pennylane/circuit_graph.py

```diff
@@ -394,20 +394,6 @@ def nodes_between(self, a, b):
         B.add(b)
         return A & B
 
-    def invisible_operations(self):
```

Comment:
Added in https://github.com/PennyLaneAI/pennylane/pull/488, but not used anymore.

Reply:
does it not require any deprecation?